docs/mutations.md

```diff
@@ -424,37 +520,29 @@ coefficients), see
 [the pyslim documentation](<https://pyslim.readthedocs.io/en/latest/>).
 For instance,
 
-```{code-block} python
+```{code-cell} python
 
 model = msprime.SLiMMutationModel(type=1)
```

Comment:
Can drop the ``discrete_genome=True`` params here and elsewhere - these are left over from when discrete wasn't the default.